be/src/storage/lake/compaction_policy.cpp

```diff
@@ -134,6 +134,8 @@ class PrimaryCompactionPolicy : public CompactionPolicy {
     StatusOr<std::vector<RowsetPtr>> pick_rowsets() override;
     StatusOr<std::vector<RowsetPtr>> pick_rowsets(const std::shared_ptr<const TabletMetadataPB>& tablet_metadata,
                                                   bool calc_score, std::vector<bool>* has_dels);
+    StatusOr<std::vector<int64_t>> pick_rowset_indexes(const std::shared_ptr<const TabletMetadataPB>& tablet_metadata,
```

Comment:
Add some comments about what does this function return, and why we need this.

Reply:
done